test/integration/targets/connection_aws_ssm/aws_ssm_integration_test_setup_teardown/tasks/main.yml

```diff
@@ -0,0 +1,106 @@
+---
+# tasks file for ec2
+
```

Comment:
Add step to install session manager plugin on your local. My bad, I forgot to tell you this. Sorry for that.

Reply:
updated and added sm plugin local installation step